src/components/ConfirmationModal.tsx

```diff
@@ -39,56 +39,48 @@ const Message = styled(HeadingText)`
 
 export interface ConfirmationModalProps {
```

Comment:
If the purpose of this component is not longer to show a confirmation-question which can be answered with "oke" or "cancel", the naming of this component should no longer be `ConfirmationModal`

Reply:
ok we can use CustomModal